core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -540,6 +540,15 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
      */
     addEditorPageScript(rendered = ''): string {
         const scriptString = `<script src="/html/js/editor-js/sdk-editor.esm.js"></script>`;
```

Comment:
Use a constant for the name of the file, maybe can change and will broke all.

Reply:
Done!